### DISCUSSION TITLE
Migration from version 2 to 4

### DISCUSSION THREAD

**Discussion by ghost** [coreui/coreui-vue#194] [Q&A]
In our frontend project, we are currently using CoreUI version 2 and are in process of migrating to CoreUI version 4 and from VueJS 2 to version 3.

Do you recommend migrating directly from CoreUI v2 to CoreUI v4, or is it better to go step by step - so to first migrate from version 2 to version 3, and then do a migration from version 3 to version 4?

Thanks! :)

**Comment by mrholek** [comment]
I recommend to migrate to v4 because this version based on Vue 3

**Reply by ghost** [reply]
ok thank you, we are working on it. Is there maybe a migration guide available? :)
